src/index.tsx

```diff
@@ -80,8 +81,14 @@ export function Playground({
   code,
   height = '200px',
   id,
+  wrappingComponent: WrappingComponent,
```

Comment:
I would rather have the `wrappingComponent` prop use pascal casing (capital "W"). I don't know if there's a convention around props being camel cased but I have not had a problem pascal casing props that are components.
```suggestion
  WrappingComponent,
```